karapace/schema_reader.py

```diff
@@ -315,6 +319,14 @@ def _is_ready(self) -> bool:
     def highest_offset(self) -> int:
         return max(self._highest_offset, self._offset_watcher.greatest_offset())
 
+    def ready(self) -> bool:
```

Comment:
I might be wrong, but as this is not a `property` object, I guess it is the counterpart of `set_not_ready()`, if so, maybe we should also set `self._ready = True` before returning it.